pinot-query-planner/src/test/java/org/apache/pinot/query/QueryCompilationTest.java

```diff
@@ -65,44 +58,6 @@ public void testQueryWithException(String query, String exceptionSnippet) {
     }
   }
 
-  @Test
```

Comment:
Why delete ?

Reply:
these test are moved to QueryPlannerSpecialfeatureTest so that we have special assert for these tests